packages/cli/bin/nuxt-cli.js

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env node
 
-require('../dist/cli.js').run()
+require('../dist/cli.js').run().then()
```

Comment:
Why this is required?

Reply:
@pi0 oh, I assumed it would be -- but you're right, Node won't exit until the last promise has finished running.

Reply:
Actually, it would be better to ensure process exit here too. Like nuxt/opencollective#4

Reply:
@pi0 indeed -- this has happened to me before, totally forgot about it.